lib/es6-promise/-internal.js

```diff
@@ -66,7 +66,8 @@ function handleForeignThenable(promise, thenable, then) {
 function handleOwnThenable(promise, thenable) {
   if (thenable._state === FULFILLED) {
     fulfill(promise, thenable._result);
-  } else if (promise._state === REJECTED) {
+  } else if (thenable._state === REJECTED) {
+    thenable._onError = null;
```

Comment:
I had no idea why it was there but it is in the rsvp commit.

Should we add a comment on why the impl is different from rsvp ?